extensions-contrib/opencensus-extensions/pom.xml

```diff
@@ -35,6 +35,10 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
+  <properties>
+    <opentelemetry.proto.version>0.11.0-alpha</opentelemetry.proto.version>
```

Comment:
we should align this with the version in the opentelemetry-extension, and only have a single property (or somehow pull this is via the dependency we define)

Reply:
make sense! that would manage the dependency in different modules in a cleaner way. Addressed all the dependency related comments.